packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,10 @@
 
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Guessing there permissions were not needed with the old library, or was this part of the problem?

Reply:
don't think so but these were in the install instructions for the new pkg!